pkg/web/apps/apps_repository.go

```diff
@@ -0,0 +1,11 @@
+package apps
+
+import (
+	"github.com/aerogear/mobile-security-service/pkg/models"
+	"github.com/labstack/echo"
+)
+
+// Repository represent the app's repository contract
+type Repository interface {
+	GetApps(c echo.Context) (*[]models.App, error)
```

Comment:
the request context should not be passed to the `Repository` layer. You should get the information you need (business object) for the request at the web layer, and from there on, you shouldn't use the request context anywhere else.

Reply:
@wei-lee Thank you Wei, we are aware of this and this is gettting changed in Austins PR here - https://github.com/aerogear/mobile-security-service/blob/473aaadfe303f0a2a7eb776cdb1797fdb9368ceb/pkg/web/apps/apps_service.go#L40

Reply:
Thanks @wei-lee . This was being tackled in a separate PR but after talking to Austin, I will make the change here

Reply:
Changed now